bind/gencffi_type.go

```diff
@@ -68,7 +85,36 @@ func (g *cffiGen) genTypeIAdd(sym *symbol) {
 	g.wrapper.Printf("\n")
 }
 
-// genTypeInit generates Type __init__.
+// genTypeGetItem generates __getitem__ of a type.
+func (g *cffiGen) genTypeGetItem(sym *symbol) {
+	g.wrapper.Printf("def __getitem__(self, idx):\n")
+	g.wrapper.Indent()
+	g.wrapper.Printf("if idx >= len(self):\n")
+	g.wrapper.Indent()
+	g.wrapper.Printf("raise IndexError('array index out of range')\n")
```

Comment:
would it be possible to raise `"array index ..."` when it's a Go array and `"slice index ..."` when it's a Go slice ?